frontend/src/component/changeRequest/ChangeRequest/Changes/Change/StrategyChange.tsx

```diff
@@ -187,34 +187,48 @@ export const StrategyChange: VFC<{
                 </>
             )}
             {change.action === 'deleteStrategy' && (
-                <ChangeItemWrapper>
-                    <ChangeItemInfo>
-                        <Typography
-                            sx={(theme) => ({
-                                color: theme.palette.error.main,
-                            })}
-                        >
-                            - Deleting strategy:
-                        </Typography>
-                        {hasNameField(change.payload) && (
-                            <StrategyTooltipLink change={change}>
-                                <StrategyDiff
-                                    change={change}
-                                    currentStrategy={currentStrategy}
-                                />
-                            </StrategyTooltipLink>
-                        )}
-                    </ChangeItemInfo>
-                    <div>{actions}</div>
-                </ChangeItemWrapper>
+                <>
+                    <ChangeItemCreateEditDeleteWrapper className='delete-strategy'>
```

Comment:
Can we have a more specific className? I can see us creating a similar className in the future and causing conflicts.

Reply:
Sure can! Let me see what I can cook up

Reply:
Addressed in 938d21663ee9faed3f9487012c4c9e5906853bb4